### PR TITLE
[CSDM-1059] add tabs enabler plugin

### PR DESCRIPTION
**Pull request by FCanberk**
Add plugin to enable tabs and add the function to enable alignment tab on enzel, sparc, secom and mimas through "Enable alignment tab" button under Help > Development on the top bar.

https://delmic.atlassian.net/browse/CSDM-1059

### CODE REVIEW

File: plugins/tabs_enabler.py [hunk 1]
```diff
@@ -0,0 +1,47 @@
+# -*- coding: utf-8 -*-
+"""
+Created on 10 April 2017
```
Comment (tepals) [inline plugins/tabs_enabler.py:3]: that's a long time ago :p

File: plugins/tabs_enabler.py [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+
+@author: Canberk Akın
+
+Gives ability to automatically place a EM image so that it's aligned with
+another one (already present).
```
Comment (tepals) [inline plugins/tabs_enabler.py:8]: I think this needs to be updated

**Review by tepals — CHANGES_REQUESTED** [review]

File: plugins/tabs_enabler.py [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+You should have received a copy of the GNU General Public License along with Odemis. If not,
+see http://www.gnu.org/licenses/.
+"""
+
+from odemis.gui.plugin import Plugin
+
+class TabEnablerPlugin(Plugin):
+    name = "Tab Enabler"
+    __version__ = "1.0"
+    __author__ = u"Canberk Akın"
```
Comment (pieleric) [inline plugins/tabs_enabler.py:29]: You can (now) drop the `u` from the string.

File: plugins/tabs_enabler.py [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+    __license__ = "Public domain"
+
+    def __init__(self, microscope, main_app):
+        super(TabEnablerPlugin, self).__init__(microscope, main_app)
```
Comment (pieleric) [inline plugins/tabs_enabler.py:33]: Now, you can just write:
```suggestion
        super().__init__(microscope, main_app)
```

File: plugins/tabs_enabler.py [hunk 5]
```diff
@@ -0,0 +1,46 @@
...
+        self.addMenu("Help/Development/Enable alignment tab", self.enable_alignment_tab)
```
Comment (pieleric) [inline plugins/tabs_enabler.py:34]: Please only add this menu if it's possible to click this option. Especially, the viewer (which is also available on all the computers, and doesn't connect to the back-end). The simplest way is probably to do the same check as in `enable_alignment_tab()`: check whether `btn_tab_align_enzel` or `btn_tab_align` are shown.

**Review by pieleric — CHANGES_REQUESTED** [review]

**Comment by pieleric** [comment]
Could you please follow the new convention for naming the PRs, and add the Jira task ID in the PR name.

**Review by pieleric — APPROVED** [review]

**Review by tepals — APPROVED** [review]

**Pull request merged**